libraries/botbuilder-ai/src/qnaMakerDialog.ts

```diff
@@ -115,8 +115,18 @@ export interface QnAMakerDialogConfiguration extends DialogConfiguration {
  * It supports knowledge bases that include follow-up prompt and active learning features.
  * The dialog will also present user with appropriate multi-turn prompt or active learning options.
  */
-export class QnAMakerDialog extends WaterfallDialog implements QnAMakerDialogConfiguration {
+export class QnAMakerDialog extends WaterfallDialog {
     public static $kind = 'Microsoft.QnAMakerDialog';
+
+    /**
+     * Log personal information flag.
+     *
+     * @remarks
+     * Defauls to a value of `=settings.logPersonalInformation`, which retrieves
+     * `logPersonalInformation` flag from settings.
+     */
+    public logPersonalInformation = new BoolExpression('=settings.logPersonalInformation');
```

Comment:
simply set to `settings.logPersonalInformation`, since `.env`, the default config file we use in our samples, doesn't support complex values unlike as appsetting in dotnet 

.env 
:x:
```
telemetry={ 
   logPersonalInformation:  true
}
```
✔️ 
```
logPersonalInformation=true
```